easybuild/toolchains/nompi.py

```diff
@@ -0,0 +1,42 @@
+##
+# Copyright 2016-2016 Ghent University
+# Copyright 2016-2016 Forschungszentrum Juelich
+#
+# This file is part of EasyBuild,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://vscentrum.be/nl/en),
+# Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# http://github.com/hpcugent/easybuild
+#
+# EasyBuild is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# EasyBuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+##
+"""
+EasyBuild support for nompi compiler toolchain (includes NVHPC and OpenMPI, and CUDA as dependency).
+
+@author: Damian Alvarez (Forschungszentrum Juelich)
+"""
+
+from easybuild.toolchains.nvhpc import NvhpcToolchain
+# We pull in MPI and CUDA at once so this maps nicely to HMNS
+from easybuild.toolchains.mpi.openmpi import OpenMPI
+from easybuild.toolchains.compiler.cuda import Cuda
+
+
+# Order matters!
+class Ompi(NvhpcToolchain, Cuda, OpenMPI):
+    """Compiler toolchain with NVHPC and OpenMPI, with CUDA as dependency."""
+    NAME = 'nompi'
```

Comment:
`nompi` easily reads as "no MPI", which is bound to lead to confusion.

I would rename this to `nvompi`

Reply:
Also, since this includes `CUDA`, should it be `nvompic`, or is that implied by the `nv` part?

Reply:
For consistency with all other toolchains it should indicate the cuda part using nvompic.
But it might be non-trivial to create a nvompi that is totally cuda clean due to nvhpc containing it already.